lib/SearchAndSort/SearchAndSortQuery.js

```diff
@@ -111,6 +111,7 @@ class SearchAndSortQuery extends React.Component {
     queryGetter: PropTypes.func,
     querySetter: PropTypes.func,
     queryStateReducer: PropTypes.func,
+    retainInternalQueryState: PropTypes.bool,
```

Comment:
Since this only affects the `search` portion of the queryState, I suggest renaming this to `retainInternalSearchState`.

Reply:
Oh of course! Tweaked name now